sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -271,7 +271,7 @@ case class AlterTableUnsetPropertiesCommand(
       }
     }
     // If comment is in the table property, we reset it to None
-    val tableComment = if (propKeys.contains("comment")) None else table.properties.get("comment")
+    val tableComment = if (propKeys.contains("comment")) None else table.comment
```

Comment:
Does this work for Hive table? When CatalogTable's comment is None, we will ignore it when converting it to Hive table metadata. So seems it will not be unset?

https://github.com/apache/spark/blob/9d0db5a7f8f0168c3cd95994b10511f8d6a241c3/sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala#L935